### PR TITLE
PTT 看板清單

### PR DESCRIPTION
**Pull request by jerry80409**
## Description
* 讓 SLACK bot 支援 PTT 看板清單, 範例：
```
/news list
```
* 新增了 [BlocksResponseRouter](https://github.com/b2etw/event-2020-1-java/blob/d16dec4558122fae780d4370497cab35dabe2d87/src/main/java/tw/b2e/common/BlocksResponseRouter.java) 介面, 支援 reply `Block` 訊息功能, 可以參考 https://api.slack.com/block-kit 
* 預覽
image

# Checklist:
- [x] My code follows the style guidelines of this project (IntelliJ Default)
- [x] My changes generate no new warnings
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/main/java/tw/b2e/news/CommandService.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      pttCommand.setMainOption(commands[0]);
+      pttCommand.setSubOperation(commands[1]);
+      return pttCommand;
+    }
+
+    if (2 < commands.length) {
+      pttCommand.setMainOption(commands[0]);
+      pttCommand.setSubOperation(commands[1]);
+      pttCommand.setArgs(
+          Arrays.asList(Arrays.copyOfRange(commands, 2, commands.length)));
```
Comment (Jian-Min-Huang) [inline src/main/java/tw/b2e/news/CommandService.java]: 你長度設多短啊，這個怎麼會換行了XDD

File: src/main/java/tw/b2e/news/PttCrawlerService.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+@Slf4j
+@Service
+public class PttCrawlerService {
+
+  // todo - I will refactor it. 人肉爬蟲 XD
```
Comment (Jian-Min-Huang) [inline src/main/java/tw/b2e/news/PttCrawlerService.java:16]: 大家都看到惹

File: src/main/java/tw/b2e/news/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+#### 列出看板最新 10 篇文章與連結, 且熱度 ( >n )
+```
+/news list [board] hot
+```
+
+---
+## TODO 
+有考慮想做的東西
+* 有支援 RSS 技術日報類 (灣區日報, 碼天狗... etc.)
+* COVID-19 趨勢與資訊
```
Comment (Jian-Min-Huang) [inline src/main/java/tw/b2e/news/README.md]: 這個要不要考慮移去外面，放在source code裡面頗怪

**Review by Jian-Min-Huang — CHANGES_REQUESTED** [review]
我覺得你寫code品味蠻好的

File: src/main/java/tw/b2e/news/PttCrawlerService.java (hunk 2 above)
Comment (jerry80409) [reply src/main/java/tw/b2e/news/PttCrawlerService.java:16]: 😂

**Comment by jerry80409** [comment]
@Jian-Min-Huang 謝謝, 我還在努力學習中...

File: src/main/java/tw/b2e/news/README.md (hunk 3 above)
Comment (jerry80409) [reply src/main/java/tw/b2e/news/README.md]: README.md 的部分可否協助我整理, 我不太確定整合到 README.md 的排版, 麻煩了, 感謝

File: src/main/java/tw/b2e/news/CommandService.java (hunk 1 above)
Comment (jerry80409) [reply src/main/java/tw/b2e/news/CommandService.java]: 自己的壞習慣 XD

**Pull request merged**